toolchain/check/diagnostic_helpers.h

```diff
@@ -14,10 +14,11 @@ namespace Carbon::Check {
 
 // Diagnostic locations produced by checking may be either a parse node
```

Comment:
Is this comment still correct, talking about parse nodes? Maybe it could explain when to use or not use this class?

Reply:
Yes, it's correct -- diagnostics are based on parse nodes. Added some notes about use.

Reply:
Oh, I see, LocId is really a parse node (or import id) - hadn't really understood that before.